src/SqlVirtualMachine/examples/New-AzSqlVM.md

```diff
@@ -20,3 +21,146 @@ Location	Name		ResourceGroupName
 eastus		sqlvm1		ResourceGroup01	
 ```
 
+Create a SQL virtual machine with specific sku type and license type.
+
+### Example 3
+```powershell
+New-AzSqlVM -ResourceGroupName 'ResourceGroup01' -Name 'sqlvm1' -Location 'eastus' -LicenseType 'AHUB'
+```
+
+```output
+Location	Name		ResourceGroupName
+--------	----		-----------------
+eastus		sqlvm1		ResourceGroup01	
+```
+
+Create a SQL virtual machine with AHUB billing tag.
+
+### Example 4
+```powershell
+New-AzSqlVM -ResourceGroupName 'ResourceGroup01' -Name 'sqlvm1' -Location 'eastus' -LicenseType 'DR'
+```
+
+```output
+Location	Name		ResourceGroupName
+--------	----		-----------------
+eastus		sqlvm1		ResourceGroup01	
+```
+
+Create a SQL virtual machine with DR billing tag.
+
+### Example 5
+```powershell
+New-AzSqlVM -ResourceGroupName 'ResourceGroup01' -Name 'sqlvm1' -Location 'eastus' -AdditionalFeatureServerConfigurationIsRServicesEnabled
+```
+
+```output
+Location	Name		ResourceGroupName
+--------	----		-----------------
+eastus		sqlvm1		ResourceGroup01	
+```
+
+Create a SQL virtual machine with Enable R services in SQL2016 onwards.
+
+### Example 6
+```powershell
+New-AzSqlVM -ResourceGroupName 'ResourceGroup01' -Name 'sqlvm1' -Location 'eastus' -AutoBackupSettingEnable `
+-AutoBackupSettingBackupScheduleType manual -AutoBackupSettingFullBackupFrequency Weekly -AutoBackupSettingFullBackupStartTime 5 ` 
+-AutoBackupSettingFullBackupWindowHour 2 -AutoBackupSettingStorageAccessKey 'keyvalue' -AutoBackupSettingStorageAccountUrl ` 
+'https://storagename1.blob.core.windows.net/' -AutoBackupSettingRetentionPeriod 10 -AutoBackupSettingLogBackupFrequency 60 ` 
+-AutoBackupSettingStorageContainerName 'storagecontainer1'
+```
+
+```output
+Location	Name		ResourceGroupName
+--------	----		-----------------
+eastus		sqlvm1		ResourceGroup01	
+```
+
+Create a SQL virtual machine and configure auto backup settings.
+
+### Example 7
+```powershell
+New-AzSqlVM -ResourceGroupName 'ResourceGroup01' -Name 'sqlvm1' -Location 'eastus' -AutoPatchingSettingDayOfWeek Thursday ` 
+-AutoPatchingSettingMaintenanceWindowDuration 120 -AutoPatchingSettingMaintenanceWindowStartingHour 3 -AutoPatchingSettingEnable
+```
+
+```output
+Location	Name		ResourceGroupName
+--------	----		-----------------
+eastus		sqlvm1		ResourceGroup01	
+```
+
+Create a SQL virtual machine and configure auto patching settings.
+
+### Example 8
+```powershell
+$password = "P@ssw0rd"
```

Comment:
This was also removed from Portal experience. Better to check with PM team/ Amitabh if we want this in the examples.